dash_cytoscape/metadata.json

```diff
@@ -349,6 +349,105 @@
           "computed": false
         }
       },
+      "contextMenu": {
+        "type": {
+          "name": "arrayOf",
+          "value": {
+            "name": "shape",
+            "value": {
+              "id": {
+                "name": "string",
+                "required": true
+              },
+              "content": {
+                "name": "string",
+                "required": true
+              },
+              "tooltipText": {
+                "name": "string",
+                "required": false
+              },
+              "coreAsWell": {
+                "name": "string",
+                "required": false
+              },
+              "selector": {
+                "name": "string",
+                "required": false
+              },
+              "onClick": {
+                "name": "union",
+                "value": [
+                  {
+                    "name": "string"
+                  },
+                  {
+                    "name": "custom",
+                    "raw": "PropTypes.function"
```

Comment:
Now that these properties have different names than the property referring to the function itself, you can remove the `function` type

Reply:
(whoops, realized this comment should be made on the js code, disregard!)